daffodil-test/src/test/resources/org/apache/daffodil/section07/escapeScheme/escapeScheme.tdml

```diff
@@ -314,6 +314,24 @@
         escapeKind="escapeCharacter" escapeEscapeCharacter="{ xs:string(xs:int(fn:ceiling(xs:float(../tns:escapeEscapeChar)))) }" extraEscapedCharacters="" generateEscapeBlock="whenNeeded" />
     </dfdl:defineEscapeScheme>
 
+    <dfdl:defineEscapeScheme name="fromData4">
+      <dfdl:escapeScheme
+              escapeCharacter="{ xs:string(xs:int(../tns:escapeChar) - 3) }"
+              escapeKind="escapeCharacter"
+              escapeEscapeCharacter="{ xs:string(xs:int(fn:ceiling(xs:float(../tns:escapeEscapeChar)))) }"
+              extraEscapedCharacters="{ xs:string(../tns:extraEscapedChars) }"
+              generateEscapeBlock="whenNeeded" />
+    </dfdl:defineEscapeScheme>
+
+    <dfdl:defineEscapeScheme name="fromData5">
+      <dfdl:escapeScheme
+              escapeCharacter="{ xs:string(xs:int(../tns:escapeChar) - 3) }"
+              escapeKind="escapeCharacter"
+              escapeEscapeCharacter="{ xs:string(xs:int(fn:ceiling(xs:float(../tns:escapeEscapeChar)))) }"
+              extraEscapedCharacters="1 3"
+              generateEscapeBlock="whenNeeded" />
+    </dfdl:defineEscapeScheme>
+
```

Comment:
It might be useful to also have a test that makes sure extraEC works with escapeKind="block". By code inspection it's clear it does, but that would prevent us accidentally breaking it in the future.